README.md

```diff
@@ -67,46 +69,77 @@ Optional:
 
 To install Auto-GPT, follow these steps:
 
-0. Make sure you have all the **requirements** above, if not, install/get them.
+1. Make sure you have all the **requirements** above, if not, install/get them.
 
-*The following commands should be executed in a CMD, Bash or Powershell window. To do this, go to a folder on your computer, click in the folder path at the top and type CMD, then press enter.*
+   *The following commands should be executed in a CMD, Bash or Powershell window. To do this, go to a folder on your computer, click in the folder path at the top and type CMD, then press enter.*
 
-1. Clone the repository:
+2. Clone the repository:
 For this step you need Git installed, but you can just download the zip file instead by clicking the button at the top of this page ☝️
-```
-git clone https://github.com/Torantulino/Auto-GPT.git
-```
 
-2. Navigate to the project directory:
-*(Type this into your CMD window, you're aiming to navigate the CMD window to the repository you just downloaded)*
-```
-$ cd 'Auto-GPT'
-```
+    ```bash
+    git clone https://github.com/Torantulino/Auto-GPT.git
+    ```
+
+3. Navigate to the project directory:
 
-3. Install the required dependencies:
-*(Again, type this into your CMD window)*
+    ```bash
+    cd 'Auto-GPT'
+    ```
+
+4. Install the required dependencies:
+
+    (**NOTE: You can skip this step if you'd rather run Auto-GPT in a Docker container (see below) instead
+    of installing its python requirements and running it locally.**)
+
+    ```bash
+    pip install -r requirements.txt
+    ```
+
+5. Copy `.env.template` to `.env` and fill in your `OPENAI_API_KEY`. If you plan to use Speech Mode, fill in your `ELEVEN_LABS_API_KEY` as well.
```

Comment:
```suggestion
5. Rename `.env.template` to `.env` and fill in your `OPENAI_API_KEY`. If you plan to use Speech Mode, fill in your `ELEVEN_LABS_API_KEY` as well.
```

Reply:
Why rename? If you rename it, then `git status` shows a missing file, but copying it and changing it works correctly, since `.env` is in the `.gitignore` file.